GruntFile.js

```diff
@@ -17,18 +17,27 @@ module.exports = function (grunt) {
             commitMessage: 'Release <%= version %>', //default: 'release <%= version %>'
             tagMessage: 'Tag version <%= version %>' //default: 'Version <%= version %>'
          }
+      },
+      
+      copy: {
+        typings: {
+            files: [
+                { expand: true, cwd: './typings', src: ['./**'], dest: '.' }
```

Comment:
Yes just to have them for NPM. I didn't just do them there so i dont bloat up your root and have the types roaming. Since i need to have `simple-git/promise`, it requires that the index is within root and the folder is relative